src/component/parent/index.js

```diff
@@ -561,7 +561,11 @@ export class ParentComponent extends BaseComponent {
 
             [ POST_MESSAGE.CLOSE ](source, data) {
 
-                this.close();
+                this.iframe.className = 'xcomponent-lightbox bounceOutDown';
+
+                setTimeout((() => {
```

Comment:
We need to think carefully about this one. `setTimeout` should really be avoided since in a lot of cases (especially in IE) if the window isn't focused, it will hang forever.

We also probably don't want to hard code a 1000ms delay here, since this bounce-out behavior is pretty specific to only one of our components.

I don't know what the ideal solution is here, but I'll have a think about it. In the mean time maybe we can just remove this timeout.